x-pack/plugin/eql/qa/common/src/main/resources/additional_test_queries.toml

```diff
@@ -273,20 +273,6 @@ file where file_name : ("winini?.exe", "lsass.e?e") and opcode == 2
 '''
 expected_event_ids  = [65, 86]
 
-[[queries]]
-name = "wildcardFunctionWildcardPattern"
-query = '''
-file where wildcard(file_name, "winini*.exe", "lsass.*") and opcode == 2
```

Comment:
should this be replaced with `like`?

Reply:
There's already a `like` test for this (`likeMultipleArgWithPattern`).